app/contact/views.py

```diff
@@ -0,0 +1,108 @@
+from flask import render_template, redirect, url_for, abort, flash
+from flask.ext.login import login_required
+from flask.ext.rq import get_queue
+
+from wtforms.fields import SelectField
+
+from .. import db
+from ..models import EditableHTML, Resource, ContactCategory
+from . import contact
+from forms import ContactForm, ContactCategoryForm, EditCategoryNameForm
+from ..email import send_email
+
+@contact.route('/', methods=['GET', 'POST'])
+def index():
+    editable_html_obj = EditableHTML.get_editable_html('contact')
+    setattr(ContactForm,
+            'category',
+            SelectField('Category', choices=[(c.name, c.name) for c in ContactCategory.query.all()]))
+    form = ContactForm()
+    contact_email = 'maps4all.team@gmail.com'
+    if form.validate_on_submit():
+        get_queue().enqueue(
+            send_email,
+            recipient=contact_email,
+            subject=form.category.data,
+            template='contact/email/contact',
+            name=form.name.data,
+            email=form.email.data,
+            message=form.message.data
+        )
+        return redirect(url_for('main.index'))
+    category_form = ContactCategoryForm()
+    if category_form.validate_on_submit():
+        new_category = ContactCategory(name=category_form.name.data)
+        db.session.add(new_category)
+        db.session.commit()
+    categories = ContactCategory.query.all()
+    return render_template('contact/index.html',
+                           editable_html_obj=editable_html_obj,
+                           form=form,
+                           category_form=category_form,
+                           categories=categories)
+
+@contact.route('/<int:category_id>', methods=['GET', 'POST'])
+@login_required
+def edit_category_name(category_id):
+    """Edit a category"""
+    category = ContactCategory.query.get(category_id)
+    if category is None:
+        abort(404)
+    old_name = category.name
+    form = EditCategoryNameForm()
+    if form.validate_on_submit():
+        if ContactCategory.query.filter(ContactCategory.name == form.name.data).first() is not None:
+            flash('Category {} already exists.'.format(form.name.data), 'form-error')
+            return render_template(
+                                'contact/manage_category.html',
```

Comment:
nit: put name of template on same line as `render_template` and then align variables with that

same for other `render_template` functions in file